### DISCUSSION TITLE
Trying to create sample application for logs

### DISCUSSION THREAD

**Discussion by vinayakadj** [open-telemetry/opentelemetry-cpp#1716] [Q&A]
I have installed opentelemetry-cpp using `vcpkg install opentelmetry-cpp`. I am using cmake for build my cmake looks like this

```
add_library(opentelemetry_imp_library opentelemetry_imp.h opentelemetry_imp.cc)
find_package(opentelemetry-cpp config REQUIRED )
find_package(CURL  config REQUIRED)
find_package(Thrift config REQUIRED )
find_package(ZLIB REQUIRED )
find_package(OpenSSL REQUIRED)
add_definitions(-DWITH_LOGS_PREVIEW) 

target_compile_definitions(opentelemetry_imp_library PRIVATE WITH_LOGS_PREVIEW=1)

target_link_libraries(opentelemetry_imp_library PUBLIC ${CMAKE_THREAD_LIBS_INIT} PRIVATE opentelemetry-cpp::api opentelemetry-cpp::ext opentelemetry-cpp::sdk opentelemetry-cpp::logs  opentelemetry-cpp::trace opentelemetry-cpp::jaeger_trace_exporter )
```
I get this warnings

Target "opentelemetry_imp_library" links to:

    opentelemetry-cpp::logs

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.
Also see lot of error in code using logs related functions when compiled with visual studio, can any body please help me why this error is seen. Code works fine with traces, but not with logs

**Comment by lalitb** [comment]
Logs implementation is still experimental (https://opentelemetry.io/docs/instrumentation/cpp/#status-and-releases), so the opentelemetry-cpp vcpkg doesn't build it by default. 
What errors are you seeing while building with VS?

**Reply by vinayakadj** [reply]
Error got solved after I installed opentelemetry-cpp[jaeger]
